views/movies/detailed.php

```diff
@@ -0,0 +1,14 @@
+
+<script> const movie_id = '<?php echo $movie_id; ?>'; </script>
```

Comment:
Doing this makes your UI code too dependent on the server side code. How can you fix this?

Reply:
Make JavaScript get it from url directly?